ldk/javascript/src/whisper/types.ts

```diff
@@ -46,6 +46,10 @@ export enum WhisperComponentType {
    * The password input field allows the user to provide a password. This field protects the user by obscuring what they type. Showing each character as a solid black dot.
    */
   Password = 'password',
+  /**
+   * The progress field allows the user to see the visualized progress indicator.
```

Comment:
For this description, maybe we should take some inspiration from the [material ui component ](https://mui.com/components/progress/) it's based off of.

It could read "Progress components express an unspecified wait time or display the length of a process."